### ISSUE TITLE
Staging Review finding: Buttons and headings aren't unique

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82855]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Pension benefits
**Product:** 527EZ
**Feature:** Update multi-response (single page list loop) pattern

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/review-and-submit](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/review-and-submit), each instance of list-and-loops results in some duplicated headings and edit buttons that don't have unique accessible names.

Every peer heading (eg. H3s nested under the same H2) should be unique. Instead, you're getting some duplication for:
1. Heading for the yes/no trigger question to start a list-and-loop and heading for the collection of items in the list
2. Headings for each item added in the list-and-loop.

The duplication for (2) might be unavoidable with how this form is structured, and even if it's not ideal I don't think the impact is that big.

The duplication for (1) is the bigger issue, since the heading names are used to populate the `aria-label` on the Edit buttons. Each Edit button needs to have a unique accessible name so that screen reader users can identify what the buttons map to.

Concrete example (also inclueded as a screenshot). If I've entered medical/unreimbursed expenses, there are:
- two H4 headings with text "Medical expenses and other unreimbursed expenses"
- multiple H5 headings with text "Unreimbursed Expense" for each item added
- two buttons with `aria-label="Edit Medical expenses and other unreimbersed expenses"`

The multiple H5s aren't a huge deal, because from context they're listing multiple instances of the same thing.

The two H4s are more confusing, since the same heading is being used twice for two different purposes.

The two Edit buttons with the same accessible name is the biggest impact for screen reader users, who might take multiple attempts to successfully edit their response.

<details><summary>Link, screenshot or steps to recreate</summary>

![Review page with two 'Medical expenses and other unreimbursed expenses' headings, two Edit buttons, and two 'Unreimbursed expense' headings.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/81262-Brian-01.png?raw=true)

</details>

## Recommended action

I believe if you give each page in the form flow a unique H3, that should trickle down and fix the review page. Using the medical/unreimbursed expenses example again, that could be something like:
- "Medical expenses and other unreimbursed expenses" H3 for the yes/no trigger page
- "List of medical expenses and other unreimbursed expenses" H3 for the list-and-loop page

That should provide enough clarity.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by TaiWilkin** [comment]
@ToddWebDev Noting that we were requested to update all pages with duplicate titles to a similar pattern (ie, "Current Employment" x2 becomes "Current Employment" and "List of Current Employment") - this was made clear in the staging review, but I'm not sure if it is clear from the ticket

**Comment by aplatt-coforma** [comment]
Demo of changes are shown here: https://www.loom.com/share/688b8ffa655a45bb8ebee7da29804cd0?sid=1d63583b-b398-42b2-a6cc-8a2abdcde15a
